fighthealthinsurance/settings.py

```diff
@@ -208,8 +208,13 @@ class Base(Configuration):
         "mfa.recovery.Hash",
     ]
 
+    CSRF_COOKIE_SECURE = True
     CSRF_COOKIE_HTTPONLY = False
 
+    # Redirect to HTTPS for all requests
+    # In Production we will use
+    # SECURE_SSL_REDIRECT = True
```

Comment:
Not needed, nginx sits in front and handles this (but a comment explaining that would be good)